src/pages/ProductList/ModalBuyNow/ModalBuyNow.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import '../ModalBuyNow/ModalBuyNow.scss';
+import ProductInfoBox from '../../../components/ProductInfoBox/ProductInfoBox';
+
+const ModalBuyNow = ({ isPopModal, setIsPopModal }) => {
+  const imgUrl = 'images/x-Mas.jpg';
+  const originPrice = 28000;
+  const productName = '닥터 타르트';
+  const productSubTag = '#가심비갑';
+
+  const clickRemoveModal = () => {
+    setIsPopModal(!isPopModal);
```

Comment:
```suggestion
    setIsPopModal(prev => !prev);
```

```js
setState(newState);
setState(prevState => newState);
```

setState  함수안에 인자로 콜백함수를 넣어줄 수 있습니다.
그 콜백함수는 인자로 이전의 state값을 전달받고 return 한 값이 새로운 state가 됩니다!
이전 state 값을 활용해서 새로운 state를 만들때는 위와 같은 방식도 참고해서 적용해보세요!